templates/blockonomics_error.php

```diff
@@ -0,0 +1,76 @@
+<?php
+$blockonomics = new Blockonomics;
+$order_hash = isset($_REQUEST["error_order"]) ? sanitize_text_field(wp_unslash($_REQUEST["error_order"])) : "";
+$order_id = $blockonomics->decrypt_hash($order_hash);
+
+$error_type = isset($_REQUEST["error_type"]) ? sanitize_text_field(wp_unslash($_REQUEST["error_type"])) : "unknown";
+$error_msg = isset($_REQUEST["error_msg"]) ? sanitize_text_field(wp_unslash($_REQUEST["error_msg"])) : "";
+
+?>
+
+<div class="bnomics-order-container">
+    <!-- Heading row -->
+    <div class="bnomics-order-heading">
+      <div class="bnomics-order-heading-wrapper">
+        <div class="bnomics-order-id">
+          <span class="bnomics-order-number"><?=__('Order #', 'blockonomics-bitcoin-payments')?><?php echo $order_id; ?></span>
+        </div>
+      </div>
+    </div>
+
+<?php
+    if ($error_type == "paid_amount_is_less") {
+?>
+    <!-- Payment Error -->
+    <div class="bnomics-order-error-wrapper">
+        <h3 class="warning bnomics-status-warning"><?=__('Paid order BTC amount is less than expected. Contact merchant', 'blockonomics-bitcoin-payments')?></h3>
+    </div>
+<?php
+    } else if ($error_type == "display_error") {
+?>
+    <!-- Display Error -->
+    <div id="display-error" class="bnomics-display-error">
+        <h2><?=__('Display Error', 'blockonomics-bitcoin-payments')?></h2>
+        <p><?=__('Unable to render correctly, Note to Administrator: Please enable lite mode in the Blockonomics plugin.', 'blockonomics-bitcoin-payments')?></p>
+    </div>
+<?php
+    } else if ($error_type == "address_generation_btc") {
```

Comment:
We have lots of repeat code to display errors. Let's pass the error title and error message from php to the template to try reserve the template for displaying of the errors. We are already doing this for some errors returned from new address generation.
```
<h2><?php echo $error_title; ?></h2>
<p>
    <?php echo $error_msg; ?>
</p>
```